demo/lightning/checkpoint/simulated/multiprocessing_train.py

```diff
@@ -103,15 +104,27 @@ def parse_args() -> argparse.Namespace:
                         action="store_true",
                         default=False,
                         help="Enable debug mode.")
+    parser.add_argument(
+        "--use-fsspec",
+        action="store_true",
+        default=False,
+        help=
+        ("Use the gcsfs reader/writer for communication with Google Cloud Storage. "
+         "If not specified, the custom GCS reader/writer provided by DataFlux (DF) will be used."
+         ))
     return parser.parse_args()
 
 
 class BenchmarkStrategy(FSDPStrategy):
 
-    def __init__(self, project: str, path: str, **kwargs):
+    def __init__(self, project: str, path: str, use_fsspec: bool, **kwargs):
         super().__init__(**kwargs)
-        self.writer = GCSDistributedWriter(path, project, None)
-        self.reader = GCSDistributedReader(path, project, None)
+        if use_fsspec:
+            self.reader = FF.FsspecReader(path)
```

Comment:
Is the provided path different for Fsspec vs GCSDist given that we don't pass it the project? or same path?

Reply:
Its the same path. Reader/Writer must be doing some sort of auth in lower layers. And since bucket name is unique universally, I dont think it needs project name at all.